src/Dfe.EarlyYearsQualification.Content/Services/ContentfulContentFilterService.cs

```diff
@@ -0,0 +1,104 @@
+using Contentful.Core;
+using Contentful.Core.Search;
+using Dfe.EarlyYearsQualification.Content.Constants;
+using Dfe.EarlyYearsQualification.Content.Entities;
+
+namespace Dfe.EarlyYearsQualification.Content.Services;
+
+public class ContentfulContentFilterService (
+    IContentfulClient contentfulClient) 
+    : IContentFilterService
+{
+    private readonly Dictionary<int, string> _months = new()
```

Comment:
I'll have a look at the AbbreviatedMonthNames as the values in the dictionary need to the match the values input on the spreadsheet.

Reply:
Changed to readonly and used the abbreviatedMonthNames

Reply:
I hope this wasn't a change too far -- I didn't realise the service had to match the existing data. It looks like we don't need to worry about i18n, since Welsh, Scottish and N Irish devolved governments would have their own equivalent for this service.

Let's keep an eye on this, and try to remember it if we ever encounter any problems with month names.